ansible/roles/ironic/templates/ironic-ipxe-httpd.conf.j2

```diff
@@ -0,0 +1,45 @@
+{% set server_root = '/etc/apache2' if kolla_base_distro in ['ubuntu', 'debian'] else '/etc/httpd' %}
+{% set user = 'www-data' if kolla_base_distro in ['ubuntu', 'debian'] else 'apache' %}
+
+# Apache httpd v2.4 minimal configuration
+# This can be reduced further if you remove the access log and mod_log_config
+ServerRoot "{{ server_root }}"
+ServerName {{ api_interface_address }}
+
+# Minimum modules needed
+LoadModule mpm_event_module modules/mod_mpm_event.so
+LoadModule log_config_module modules/mod_log_config.so
+LoadModule dir_module modules/mod_dir.so
+LoadModule authz_core_module modules/mod_authz_core.so
```

Comment:
Is authorisation required?

Reply:
its is required for the Require directive